src/utils/object-inspector.js

```diff
@@ -180,8 +180,21 @@ function getChildren(
   // being the same across renders. If we didn't do this, each
   // node would be a new instance every render.
   const key = item.path;
-  if (actors && actors[key]) {
-    return actors[key];
+  if (actors && actors[key] && item.contents.value.preview) {
+    let thisActor = actors[key];
+    let index = 0;
+    const properties = item.contents.value.preview.ownProperties;
+    thisActor.forEach(object => {
+      if (
+        properties[object.name] &&
+        !(properties[object.name] instanceof Function) &&
+        object.name != "__proto__"
+      ) {
+        thisActor[index].contents = properties[object.name];
+      }
+      index++;
+    });
+    return thisActor;
```

Comment:
Also , the `actors[key]` only goes stale across the `ObjectInspector` component loads, therefore across internal component renders we should still be fine doing `return actors[key]` if it exists.